src/examples/example1.obfl

```diff
@@ -71,6 +71,14 @@
 				</dynamic-sequence>
 			</post-content>
 		</volume-template>
+		<collection name="notes">
```

Comment:
This doesn't really have anything to do with the actual issue right?

Reply:
You're right. 😉
I had to make this change so that this example validates.